cmd/acra-connector/acra-connector.go

```diff
@@ -209,7 +209,7 @@ type Config struct {
 }
 
 func main() {
-	log.Infof("Starting service %v [pid=%v]", ServiceName, os.Getpid())
+	log.WithField("version", utils.VERSION).Infof("Starting service %v [pid=%v]", ServiceName, os.Getpid())
```

Comment:
Hm... Why add version as a field but keep service name and PID just a part of the log string?

Reply:
hm... I don't know what to say) legacy version vs new version)) All our components log at startup like `Starting service BLABLA [pid=123]` and only 3 (server/connector/translator) works as a daemon and good to log version for them. So used field to log instead raw string format